src/components/ChallengeEditor/index.js

```diff
@@ -1435,6 +1461,7 @@ class ChallengeEditor extends Component {
         )
       }
     </React.Fragment>
+    const useTask = _.find(metadata.challengeTypes, { id: challenge.typeId, isTask: true })
```

Comment:
`isTask` variable is already there to determine if the challenge is actually a task.